src/CommunityToolkit/Xamarin.CommunityToolkit/Views/Snackbar/SnackBar.android.cs

```diff
@@ -18,6 +18,11 @@ internal void Show(Page sender, SnackBarOptions arguments)
 		{
 			var view = Platform.GetRenderer(sender).View;
 			var snackBar = AndroidSnackBar.Make(view, arguments.MessageOptions.Message, (int)arguments.Duration.TotalMilliseconds);
+			var anchorView = arguments.AnchorView?.GetRenderer()?.View;
+			if (anchorView != null)
+			{
+				snackBar.SetAnchorView(anchorView);
+			}
```

Comment:
```suggestion
				snackBar.SetAnchorView(anchorView);
```

Reply:
Hey @AndreiMisiukevich maybe I've been doing it somewhere too, but let's prefer braces for single line ifs as well. See the link in #884 this is also suggested by our current editorconfig file. This was something that I recently learned and I think it makes sense. If you don't agree let's discuss, thanks!